website/community/release/release.md

```diff
@@ -139,11 +139,10 @@ For example:
 Bump all components' version in the project to the new opendal version.
 Please note that this version is the exact version of the release, not the release candidate version.
 
-- rust core: bump version in `Cargo.toml`
-- cpp binding: bump version in `bindings/cpp/CMakeLists.txt`
```

Comment:
Removed. I don't see such a version field anymore.

cc @PragmaTwice is this the case?

Reply:
> is this the case?

That we don't have a version field for Opendal C++ binding in CMakeLists.txt, and it's intended.

If so, I'd like to know the reason.

Reply:
It's not so common to maintain such VERSION field in `project()` in C++ project.

I think I remove it since the version number is stale. You can add it again if you want.

Reply:
> if you want.

No need :D

> not so common to maintain such VERSION

May I ask what is the common practice to version a C++ project?

Reply:
I don't think there's a common practice, it's can be just git tags, or text files, or generate by scripts, e.g. https://github.com/ClickHouse/ClickHouse/blob/8a04621f436a36b35346ff474fb980469b61647d/cmake/autogenerated_versions.txt#L1 .

Reply:
Get it.